utils/build-command-groups.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Viktor Söderqvist <viktor.soderqvist@est.tech>
```

Comment:
Is there a reason you assigned the copyright of this file to you and the other files to valkey contributors?

Reply:
Not really, just that I didn't know exactly what we prefer nowdays. "The Valkey contributors"? (The copyright belongs to the author regardless and I suspect there won't be any large contributions to these scripts.)

Reply:
My preference is using Valkey contributors for everything committed to our shared repositories just so we are standardized and consistent. You are right that the copyright still belongs to the original author. You are also right that I doubt these files will be modified, but they might be one day.

Reply:
Changed.